mod-source-record-manager-server/src/main/java/org/folio/rest/impl/MetadataProviderImpl.java

```diff
@@ -169,6 +169,22 @@ public void getMetadataProviderJobProfiles(int offset, int limit, Map<String, St
     });
   }
 
+  @Override
+  public void getMetadataProviderJobExecutionsUniqueUsers(int offset, int limit, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    vertxContext.runOnContext(v -> {
+      try {
+        jobExecutionService.getUniqueUsersInfo(offset, limit, tenantId)
+          .map(GetMetadataProviderJobExecutionsUniqueUsersResponse::respond200WithApplicationJson)
+          .map(Response.class::cast)
+          .otherwise(ExceptionHelper::mapExceptionToResponse)
```

Comment:
when an exception occurs, would it be written to the logs?

Reply:
Hi @okolawole-ebsco! Yes, this is a standard mechanism for exception catching+logging.